src/modes/edit_message.cc

```diff
@@ -95,6 +95,12 @@ namespace Astroid {
 
 # ifndef DISABLE_EMBEDDED
     embed_editor = !editor_config.get<bool> ("external_editor");
+    char const* xdg_session_type = std::getenv("XDG_SESSION_TYPE");
+    LOG (info) << "xdg_session_type: " << xdg_session_type;
```

Comment:
Change to (debug) and move into if; add else-condition for xdg_session_type NULL.